lib/mapObjects/JsonRandom.cpp

```diff
@@ -89,6 +90,26 @@ namespace JsonRandom
 	{
 		std::string resourceName = loadKey(value, rng, "");
 		si32 resourceAmount = loadValue(value, rng, 0);
+		
+		if(!value["simple"].isNull())
```

Comment:
But this is already supported without hardcoded resources list:
`"list" : [ "wood", "ore" ]`
`"list" : [ "mercury", "gems", "sulfur", "crystal" ]`

Reply:
yes, it's more like syntax sugar. If we don't need this, I'll remove

Reply:
Probably not needed.
I have an idea to introduce "traits" to objects so you can (for example) pick objects based on said trait, for example pick an artifact that has traits "relic" and "rightHand" to pick random relic sword.

In that case resources might get "common" or "rare" traits. But right now this is just an idea with rather low priority. Don't see need for it at the moment